src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA");
+        Manufacturer manufacturer2 = new Manufacturer(null, "Ford", "USA");
+        Manufacturer manufacturer3 = new Manufacturer(null, "Audi", "Germany");
+        Manufacturer manufacturer4 = new Manufacturer(null, "BMW", "Germany");
```

Comment:
1) It is best not to use numbers in the variable names, use words in letters
2) You already have another constructor with name and country params, so you can use it instead of putting null:
```suggestion
        Manufacturer manufacturer1 = new Manufacturer("Lincoln", "USA");
        Manufacturer manufacturer2 = new Manufacturer("Ford", "USA");
        Manufacturer manufacturer3 = new Manufacturer("Audi", "Germany");
        Manufacturer manufacturer4 = new Manufacturer("BMW", "Germany");
```